__tests__/app.test.js

```diff
@@ -209,4 +209,88 @@ describe("app", () => {
       });
     });
   });
+
+  describe("GET /api/articles/:article_id/comments", () => {
+    it("responds with a status 200 if successful", () => {
+      return request(app).get("/api/articles/1/comments").expect(200);
+    });
+    it("responds with an array of comments with the correct amount for the id searched", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+          let comments = res.body.comments;
+          expect(comments.length).toBe(11);
+        });
+    });
+    it("responds with an array of comment objects with the correct properties", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+          let comments = res.body.comments;
+          expect(comments).toEqual(
+            expect.arrayContaining([
```

Comment:
This should still be iterated through to check that each comment has what you're expecting.